driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -933,6 +938,90 @@ public static Bson searchMeta(final SearchCollector collector, final SearchOptio
         return new SearchStage("$searchMeta", notNull("collector", collector), notNull("options", options));
     }
 
+    /**
+     * Creates a {@code $vectorSearch} pipeline stage supported by MongoDB Atlas.
+     * You may use the {@code $meta: "vectorSearchScore"} expression, e.g., via {@link Projections#metaVectorSearchScore(String)},
+     * to extract the relevance score assigned to each found document.
+     *
+     * @param queryVector The query vector. The number of dimensions must match that of the {@code index}.
+     * @param path The field to be searched.
+     * @param index The name of the index to use.
+     * @param numCandidates The number of candidates.
+     * @param limit The limit on the number of documents produced by the pipeline stage.
+     * @return The {@code $vectorSearch} pipeline stage.
+     *
+     * @mongodb.atlas.manual atlas-vector-search/vector-search-stage/ $vectorSearch
+     * @mongodb.atlas.manual atlas-search/scoring/ Scoring
+     * @mongodb.server.release 7.1
```

Comment:
I noticed that this feature was also backported to 6.0.10 as mentioned in this Jira ticket (https://jira.mongodb.org/browse/PM-3407).

I recall a similar instance where we specified the backported version in @mongodb.server.release for [one](https://github.com/mongodb/mongo-java-driver/pull/1159/commits/019922394451b92ed8a823b913c654c916e533fd) of the previous PRs. Considering this, should we include the backported version here as well?

Reply:
Done.